cassiopeia/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java

```diff
@@ -350,4 +345,38 @@ static String getTesseractProg() {
         return System.getProperty("os.name").startsWith("Windows") ? "tesseract.exe" : "tesseract";
     }
 
+    /**
+     * This function gets the list of available languages of Tesseract.
+     * 
+     * @param propertyManager
+     * @return String[] - the list of available languages
+     */
+    public List<String> getTessLangs(IPropertiesManager propertyManager) {
```

Comment:
let's not introduce a new dependency here, just use the TesseractOCRConfig class used in the other methods.